crates/vm-superio/src/rtc_pl031.rs

```diff
@@ -134,6 +134,20 @@ pub struct Rtc<EV: RtcEvents> {
     events: EV,
 }
 
+/// The state of the Rtc device.
+pub struct RtcState {
```

Comment:
I think it might also help to have a derive Default, but I am not sure what should be the implementation of that. Maybe we can implement default ourselves and have everything as 0s expect the LR which we can initialize as the current time on host.

Reply:
Default is already implemented for RtcState. I was using `derive` initially, but switched to a manual implementation to be on the safe side, and it also helped to have a clearer documentation.

Reply:
Added derives for Debug, Clone and PartialEq. I also realized now that `RtcStateSer` was deriving Default. I removed the derive, we can add a manual implementation if we think it's useful, but it doesn't seem so, at least for now.

Reply:
I also added a `Default` impl, it would look a bit awkward to convert with `From` from an RtcState::default() in the consumer code in case a default RtcStateSer is needed.